packages/web/src/components/ui/Header/Header.tsx

```diff
@@ -6,11 +6,15 @@ import { Icon } from '../Icon/Icon';
 import { trackCustomEvent } from '../../../services/analytics';
 
 type Props = {
-  variant?: 'default' | 'homepage';
+  variant?: 'default' | 'homepage' | 'error';
```

Comment:
nit: Seems like an attempt to fit too much into a single component

Reply:
How would you do it otherwise? Their purpose here is just to slightly modify styles (spacing and which elements to show in case of error page) to match design differences between places where they are used. Is that really to much for one component/wrapper to handle?

Reply:
This relates more to correctly segregating the responsibilities, rather than amount of code or variants.

I.e.:
* Passing links to render as a render prop. This gets rid of `display: none;` hack
* The differing padding between variants might be solved by underlying content (or, rather, the parent component)